### PR TITLE
vera.md

### PR DESCRIPTION
**Pull request by chiomavera**

### CODE REVIEW

**Comment by chiomavera** [comment]
> _No description provided._

secondpr

File: firstpr/vera.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Hi,I'm Chioma Vera Nkanmuo, Beginner Web developer from Nigeria with basic knowledge of  HTML and  CSS. I am currently in my penultimate level in university.
+Aside  from coding, I love to  read alot.
+FUN FACT: I am a fitness ethusiam.
```
Comment (sarathkumar6) [inline firstpr/vera.md:3]: Minor: typo `entusiast`. 
No issues, can merge for now.

**Review by sarathkumar6 — APPROVED** [review]

**Review by michaelcosj — APPROVED** [review]
I think it's all good

File: firstpr/vera.md (hunk 1 above)
Comment (chiomavera) [reply firstpr/vera.md:3]: > Minor: typo `enthusiast`.
> No issues, can merge for now.

SarathKumar6 thanks pointing out the typo

**Review by tusharkandpal — APPROVED** [review]

**Review by sarathkumar6 — APPROVED** [review]

**Pull request merged**